### PR TITLE
FORMS-11475 Unable to select the secondary panels individually in Accordion, Wizard, Horizontal  and Vertical Tabs

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by dmaurya929** [comment]
@sakshi-arora1 @rismehta requesting to review this PR.

**Comment by dmaurya929** [comment]
@rismehta please approve this PR.

**Review by sakshi-arora1 — APPROVED** [review]

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <div data-sly-use.accordion="com.adobe.cq.forms.core.components.models.form.Panel"
      data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
-     data-panelcontainer="${wcmmode.edit && 'accordion'}"
+     data-panelcontainer="${wcmmode.edit && 'adaptiveFormAccordion'}"
```
[the remark this answers is not in the document]
Comment (sakshi-arora1) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html:18]: @rismehta This was a bug and is just modifying HTML of authouring, hence its not a backward incompatible cahnge

**Comment by dmaurya929** [comment]
@rismehta please approve if looks good.

**Review by rismehta — APPROVED** [review]

**Pull request merged**